aea/test_tools/test_cases.py

```diff
@@ -24,18 +24,23 @@
 import subprocess  # nosec
 import sys
 import tempfile
-from typing import Any, List
+from io import TextIOWrapper
+from threading import Thread
+from typing import Any, Callable, List
 
 import pytest
 
 from aea.cli import cli
-from aea.cli_gui import DEFAULT_AUTHOR
 from aea.configurations.constants import DEFAULT_REGISTRY_PATH
 from aea.test_tools.click_testing import CliRunner
+from aea.test_tools.config import (
+    AUTHOR,
+    CLI_LOG_OPTION,
+    DEFAULT_PRIVATE_KEY_FILE_NAME,
+    FETCHAI_NAME,
+)
 from aea.test_tools.exceptions import AEATestingException
 
-CLI_LOG_OPTION = ["-v", "OFF"]
```

Comment:
Throughout AEA we always declare constants in the relevant files, the config. file approach creates additional verbosity.